rest_framework/fields.py

```diff
@@ -778,6 +778,8 @@ def to_internal_value(self, data):
         return data
 
     def to_representation(self, value):
+        if value is None:
+            return None
```

Comment:
I'm fine with the 3.4 line dropping the 1.7.
I would feel better if that was part of a different PR.